src/ServiceBusChannel.php

```diff
@@ -189,7 +189,7 @@ public function generateTokenKey()
     {
         return md5(
             'service-bus-token' .
-            Arr::get($this->ventureConfig, 'venture_config_id')
+            Arr::get($this->node, 'venture_config_id')
```

Comment:
v2.0.0 doesn't have venture_config_id. use from and fallback to venture_config_id if empty, i think.